Modules/_sqlite/connection.c

```diff
@@ -1478,6 +1478,94 @@ pysqlite_connection_iterdump(pysqlite_Connection* self, PyObject* args)
 }
 
 static PyObject *
+pysqlite_connection_backup(pysqlite_Connection* self, PyObject* args, PyObject* kwds)
```

Comment:
Let's omit the whole function. This what we do for other version specific features in the stdlib. Please also update the documentation to note that this feature is available from SQLite 3.6.11.